### PR TITLE
Update howto to be more generic

### PR DESCRIPTION
**Pull request by alexjh**

### CODE REVIEW

**Comment by aarondl** [comment]
Not that I'm specifically reviewing these but following along out of curiosity I noted that it'd be easier to compress the storage class stuff into JSON and make it a one liner.

If kube 1.5:
```bash
echo '{"kind":"StorageClass","apiVersion":"storage.k8s.io/v1beta1","metadata":{"name":"hostpath"},"provisioner":"kubernetes.io/host-path","parameters":{"path":"/tmp"}}' | kubectl apply -f -
```

File: How-to-Install-SCF.md [hunk 1]
```diff
@@ -143,58 +113,97 @@ To install SCF
...
+   ```
+   cat > storage-class-host-path.yaml <<END
+   ---
+   kind: StorageClass
+   apiVersion: storage.k8s.io/v1beta1
+   metadata:
+     name: hostpath
+   provisioner: kubernetes.io/host-path
+   parameters:
+     path: /tmp
```
Comment (mook-as) [inline How-to-Install-SCF.md]: I'm pretty sure the `parameters` here doesn't actually do anything. (It was an attempt to feed it things, but it wasn't actually used.)

File: How-to-Install-SCF.md [hunk 2]
```diff
@@ -207,8 +216,11 @@ To install SCF
   To invoke the tests run the command
    ```
    kubectl create \
-      --namespace="${NAMESPACE}" \
+      --namespace=scf \
       --filename="kube/cf/bosh-task/smoke-tests.yml"
```
Comment (mook-as) [inline How-to-Install-SCF.md:200]: Don't we need things like overriding variables that are meant for vagrant (e.g. change `DOMAIN`)?

File: How-to-Install-SCF.md [hunk 3]
```diff
@@ -207,8 +216,11 @@ To install SCF
...
+
+   # Wait for completion
+   kubectl attach --namespace=scf smoke-tests
```
Comment (mook-as) [inline How-to-Install-SCF.md]: Isn't `kubectl logs -f` enough?

Comment (alexjh) [reply How-to-Install-SCF.md]: Fixed in 79937f2

File: How-to-Install-SCF.md (hunk 2 above)
Comment (alexjh) [reply How-to-Install-SCF.md:200]: Is there a way to do this other than changing the yml file? I don't see anything obvious in the docs. I guess this is why we want helm for this instead.

Comment (mook-as) [reply How-to-Install-SCF.md:200]: Nope. (Unless you count doing it with pipes and `kubectl create -f -` as not changing the file.)

Comment (alexjh) [reply How-to-Install-SCF.md:200]: Fixed in e1afe0b

File: How-to-Install-SCF.md (hunk 1 above)
Comment (alexjh) [reply How-to-Install-SCF.md]: Fixed in 1900783

**Comment by alexjh** [comment]
Turned the StorageClass setup into a JSON one-liner. 1900783

**Pull request closed without merging**